101-cosmosdb-create/README.md

```diff
@@ -0,0 +1,21 @@
+# Create an Azure Cosmos DB account for Core (SQL) API, a database, and a container 
+
```

Comment:
add badges (see ci results)

Reply:
Aren't the labels/badges automatically added?